SimTKcommon/include/SimTKcommon/internal/Pathname.h

```diff
@@ -87,6 +87,26 @@ namespace SimTK {
  */
 class SimTK_SimTKCOMMON_EXPORT Pathname {
 public:
+    /// Given a specified working directory (swd) and path, this function
+    /// evaluates the absolute path of a given path relative to the swd and
+    /// returns the directory, fileName, and extension of the canonicalized absolute
+    /// path. This means that instead of evaluating "." as the current working
+    /// directory (cwd), the swd is used. In general, some rules are as follows
+    /// 1) If an absolute path is given by path, swd is not used. An absolute path
+    ///    is defined as a a root-relative path name and on Windows it will begin 
+    ///    with an explicit drive letter (e.g. /usr/* or c:/documents/*, respectively).
+    /// 2) If a relative path without a drive letter is given by path (e.g. "./*" or "*"), 
```

Comment:
I don't think it is good to use `*` here because it looks like a glob pathname (file regular expression). It would be easy for people to misinterpret this to mean they can provide patterns (or am I worrying too much?). An alternative might be something innocuous like `xxx`.